core/src/bufr/BufrReader/Exports/Variables/RemappedBrightnessTemperatureVariable.cpp

```diff
@@ -64,6 +64,9 @@ namespace bufr {
         int nobs = (radObj->getDims())[0];
         int nchn = (radObj->getDims())[1];
 
+        log::info()  << "RemappedBrightnessTemperatureVariable: nobs = " << nobs << std::endl;
+        log::info()  << "RemappedBrightnessTemperatureVariable: nchn = " << nchn << std::endl;
+
```

Comment:
Are you sure you wanted to keep these print statements?

Reply:
@rmclaren I will remove the print statements.